partiql-test-data/pass/parser/query/select/joins.ion

```diff
@@ -29,3 +29,129 @@ parse::{
         result: ParseOk
     }
 }
+
+parse::{
```

Comment:
These tests from L33-157 were mistakenly marked as `fail` in the first commit. Moved to `pass` in 26a5720d8eba843e9f4d8421d5c25f511eefb449.